src/components/fields/input-field.vue

```diff
@@ -168,4 +182,20 @@ export default defineComponent({
     text-decoration: none;
   }
 }
+.leading-icon,
+.trailing-icon {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 12px 24px;
+}
+input[type="number"]::-webkit-inner-spin-button,
+input[type="number"]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+input[type="number"] {
+  -moz-appearance: textfield;
+}
```

Comment:
This should be wrapped by `.input-field` selector to not be bubbled to the global scope